xml/geo_booth_i.xml

```diff
@@ -64,6 +64,75 @@
    parties.
   </para>
  </sect1>
+
+ <sect1 xml:id="sec.ha.geo.booth.ticket-types">
+  <title>Automatic versus Manual Tickets</title>
+  <para>
+   A ticket grants the right to run certain resources on a specific
+   cluster site. Two types of tickets are supported:</para>
+  <itemizedlist>
+   <listitem>
+    <para>Automatic tickets are controlled by the &boothd; daemon.</para>
+   </listitem>
+   <listitem>
+    <para>Manual tickets are managed by the cluster administrator only.</para>
+   </listitem>
+  </itemizedlist>
+
+  <para>Automatic and manual tickets have the following properties:</para>
+  <itemizedlist>
+   <listitem>
+    <formalpara>
+     <title>Automatic and manual tickets can be defined together</title>
+     <para>You can define and use both automatic and manual tickets within the
+      same &geo; cluster.</para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Manual ticket management remains manual</title>
+     <para>The automatic ticket management is not applied to manually controlled
+      tickets.
+      Manual tickets do not require any quorum elections, cannot fail over
+      automatically, and do not have an expiry time.</para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Manual tickets are not moved</title>
+     <para>Tickets which were manually granted to a site will remain there
+      until they are manually revoked. Even if a site goes offline, the
+      ticket will not be moved to another site. This behavior ensures that
+      the services that depend on a ticket remain on a particular site and
+      are not moved to another site.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Same commands for managing both types of tickets</title>
+     <para>The manual tickets are managed by the same commands as automatic
+     tickets, using the <command>grant</command> or <command>revoke</command>
```

Comment:
How about changing:
"using the <command>grant</command> or <command>revoke</command> subcommands, for example."
To just:
"for example <command>grant</command> or <command>revoke</command>."